### PR TITLE
Add request_doi.py to create test DOI for warmups

### PR DESCRIPTION
**Pull request by mtorrisi**

### CODE REVIEW

File: FutureGateway/php/17_sayhello.data [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+-rw-rw-r-- 1 jobtest jobtest 108 Feb 13 13:07 sayhello.txt
+User jobtest says: I am saying hello!
+Receiving sayhello.txt file
+!!!Say hello!!!
+---------------
+Demonstrative application for APIServer
+using I/O Sandboxing and arguments
+
+waiting for a while (2 minutes) ... done
+ending at: Mon Feb 13 13:09:26 UTC 2017
```
Comment (brucellino) [inline FutureGateway/php/17_sayhello.data]: I would suggest changing the suffix of this file from `.data` to a Markdown file.

File: OAR/curl/request_doi.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+import json,cgi,time
+import httplib2, sys, base64, codecs
+
+res=[]
+retCode=0
+errCode=''
+doi='11623'
+res =  "%s/sci-gaia:%s" % (doi,time.time())
+print "Content-type: application/json\n\n"
+print json.dumps(res)
```
Comment (brucellino) [inline OAR/curl/request_doi.py]: why is the python script in the curl directory ?

File: request_doi.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+import json,cgi,time
+import httplib2, sys, base64, codecs
+
+res=[]
+retCode=0
+errCode=''
+doi='11623'
+res =  "%s/sci-gaia:%s" % (doi,time.time())
+print "Content-type: application/json\n\n"
+print json.dumps(res)
```
Comment (brucellino) [inline request_doi.py]: Please include a few comments in the code to explain what it's doing.

**Review by brucellino — CHANGES_REQUESTED** [review]
A few major changes requested.

File: FutureGateway/php/17_sayhello.data (hunk 1 above)
Comment (mtorrisi) [reply FutureGateway/php/17_sayhello.data]: I delete this file in the other commit is not needed.

File: OAR/curl/request_doi.py (hunk 2 above)
Comment (mtorrisi) [reply OAR/curl/request_doi.py]: Is just a quick way t create DOI for warm-up session.

File: request_doi.py (hunk 3 above)
Comment (mtorrisi) [reply request_doi.py]: I'll do

File: FutureGateway/php/17_sayhello.data (hunk 1 above)
Comment (brucellino) [reply FutureGateway/php/17_sayhello.data]: ok. I agree. I would add some information about how to run the python script (with some of this text in the .data file) to the README in the higher-up dir.

File: OAR/curl/request_doi.py (hunk 2 above)
Comment (brucellino) [inline OAR/curl/request_doi.py]: Ok, then just write that as a comment in the top :)

Comment (brucellino) [reply OAR/curl/request_doi.py]: Yes, but it's in the cURL dir. I would expect cURL stuff in the cURL dir and python stuff in the python dir. Can you re-write this request as a cURL request, perhaps in `../../README.md` ?

File: FutureGateway/php/17_stdout [hunk 4]
```diff
@@ -0,0 +1 @@
+sayhello.sh: line 6: fg-hackfest.sci-gaia.eu: command not found
```
Comment (brucellino) [inline FutureGateway/php/17_stdout]: Why is there reference to `sayhello.sh` in a php script ? Surely this should be `sayhello.php` ?

Comment (mtorrisi) [reply FutureGateway/php/17_stdout]: Also this file will be removed is not needed

File: OAR/curl/request_doi.py (hunk 2 above)
Comment (mtorrisi) [reply OAR/curl/request_doi.py]: Actually is not a request, it just generates a unique identifier for the object based on the timestamp. I'm trying to write a README.md in curl folder that try to better explains how to perform the warmup. What do you think?

**Comment by brucellino** [comment]
@mtorrisi still waiting for the `git rm ; git commit ; git push` ...

File: OAR/curl/request_doi.py (hunk 2 above)
Comment (brucellino) [inline OAR/curl/request_doi.py]: I think you should still  do it with cURL in this directory, not python.

Comment (brucellino) [reply OAR/curl/request_doi.py]: @mtorrisi I take back what I said before, sorry. This is ok - I would just change doi -> prefix and res->doi

**Comment by mtorrisi** [comment]
The 	5b474a7 should address the last comments

**Review by brucellino — APPROVED** [review]
You are worthy, @mtorrisi

**Pull request merged**